exec.go

```diff
@@ -33,6 +33,10 @@ following will output a list of processes running in the container:
 			Name:  "console-socket",
 			Usage: "path to an AF_UNIX socket which will receive a file descriptor referencing the master end of the console's pseudoterminal",
 		},
+		cli.StringFlag{
+			Name:  "pidfd-socket",
+			Usage: "path to an AF_UNIX socket which will receive a file descriptor referencing the init process",
```

Comment:
s/init/exec/

Reply:
Sorry for late reply. As far as I know, the `exec` is to run new process in the existing container scope.
And the code base also uses `setns_init_process.go` to take `exec` as `init` as well. 
I'm ok to update it with `exec`. Appreciate if there is more information about this

Reply:
ping @lifubang @AkihiroSuda @cyphar @kolyshkin ~